### ISSUE TITLE
[REQ] Termux Support & Guide!

### ISSUE THREAD

**Issue by DineshValor** [shamhi/HamsterKombatBot#2248]
Termux requirements and guidance for fellow Android users.

Running on Termux (Android 14)
![Screenshot_20240823-222022](https://github.com/user-attachments/assets/7bf96ff0-f9e2-4201-800d-73ad133c752f)

https://github.com/user-attachments/assets/a6fb559a-fb0c-4207-ae69-00787fcf9c9e

**Comment by semeer37** [comment]
Playwright won't work on termux(you might have to disable minigame, since it is only one key/day). But playwright works on nethunter(nano version is enough)

**Comment by DineshValor** [comment]
> I am also running on termux, but on nethunter. Playwright won't work on termux(you might have to disable minigame, since it is only one key/day). But playwright works on nethunter(nano version is enough)

Can you create a fork supporting Termux?

**Comment by semeer37** [comment]
> > I am also running on termux, but on nethunter. Playwright won't work on termux(you might have to disable minigame, since it is only one key/day). But playwright works on nethunter(nano version is enough)
> 
> Can you create a fork supporting Termux?

I did it once(I even wrote one liner installation script). They keep on updating the bot every week. I  don't have time(sorry, I am too lazy for that) to manage. So i deleted the repo.

The changes in termux are we have to install rust for pydantic, disable minigame(in .env) enable comment for import playwright. That's it.

If anyone want to run the bot as per in orginal README, just install kali nethunter rootless nano version.

Brother, if you can maintain, you do it.

**Comment by DineshValor** [comment]
> > > I am also running on termux, but on nethunter. Playwright won't work on termux(you might have to disable minigame, since it is only one key/day). But playwright works on nethunter(nano version is enough)
> > 
> > 
> > Can you create a fork supporting Termux?
> 
> I did it once(I even wrote one liner installation script). They keep on updating the bot every week. I don't have time(sorry, I am too lazy for that) to manage. So i deleted the repo.
> 
> The changes in termux are we have to install rust for pydantic, disable minigame(in .env) enable comment for import playwright. That's it.
> 
> If anyone want to run the bot as per in orginal README, just install kali nethunter rootless nano version.
> 
> Brother, if you can maintain, you do it.

Deleting a repo not an good idea, what if someone get an idea or caught any minimal mistake from other's sources.

Anyways thanx for the advice, I will try to do it, if got lot of free time.

**Comment by DineshValor** [comment]
Made a fork specially for Termux users.

https://github.com/DineshValor/HamsterValor

**Comment by socloseeee** [comment]
> ### **Made a fork specially for Termux users.**
> 
> 
> 
> https://github.com/DineshValor/HamsterValor
> 
> 
> 
> Edit:- i have migrated this to new one, search in my repo, if still not found that's mean deleted or private due to some reasons.

Did you delete the repository?

**Comment by DineshValor** [comment]
Yep, bro currently I'm not active much so I can maintain it with @shamhi
repo. So I decided to delete it.
